util/src/main/java/io/kubernetes/client/Configmaps.java

```diff
@@ -0,0 +1,197 @@
+          new BufferedReader(new FileReader(String.valueOf(file)))) {
+        create(namespace, name, bufferedReader);
+      }
+    } else {
+      throw new CreateConfigmapException("Target path is a directory");
+    }
+  }
+
+  /**
+   * To create Configmap by passing InputStream of a file which has key value pairs
+   *
+   * @param namespace
+   * @param name
+   * @param stream
+   * @throws IOException
+   * @throws CreateConfigmapException
+   * @throws ApiException
+   */
+  public void create(String namespace, String name, InputStream stream)
+      throws IOException, CreateConfigmapException, ApiException {
+    create(namespace, name, new InputStreamReader(stream));
+  }
+
+  /**
+   * To creat Configmap by passing string literal. Value should be a string in the form "key=value"
+   *
+   * @param namespace
+   * @param name
+   * @param value
+   * @throws IOException
+   * @throws CreateConfigmapException
+   * @throws ApiException
+   */
+  public void create(String namespace, String name, String value)
+      throws IOException, CreateConfigmapException, ApiException {
+    create(namespace, name, new StringReader(value));
+  }
+
+  /**
+   * To create Configmap by passing Reader of a file which has key value pairs
+   *
+   * @param namespace
+   * @param name
+   * @param reader
+   * @throws IOException
+   * @throws CreateConfigmapException
+   * @throws ApiException
+   */
+  public void create(String namespace, String name, Reader reader)
+      throws IOException, CreateConfigmapException, ApiException {
+    if (reader != null) {
+      if (name != null && !name.isEmpty()) {
+        V1ConfigMap v1ConfigMap = createConfigmapObject(name);
+        byte[] data = CharStreams.toString(reader).getBytes();
+        // populate base64 binary data in V1ConfigMap
+        Map<String, byte[]> binMap = new HashMap<>();
+        binMap.put(name, Base64.getEncoder().encode(data));
+        createConfigmap(namespace, v1ConfigMap);
+      } else {
+        throw new CreateConfigmapException("Invalid Configmap name.");
+      }
+    } else {
+      throw new CreateConfigmapException("Reader object is null.");
+    }
+  }
+
+  /**
+   * Helper method to create request object to create Configmap.
+   *
+   * @param configmapName
+   * @return
+   */
+  private V1ConfigMap createConfigmapObject(String configmapName) {
+    V1ConfigMap v1ConfigMap = new V1ConfigMap();
+    V1ObjectMeta metadata = new V1ObjectMeta();
+    metadata.setName(configmapName);
+    v1ConfigMap.setKind("ConfigMap");
+    v1ConfigMap.setApiVersion("v1");
+    v1ConfigMap.setMetadata(metadata);
+    return v1ConfigMap;
+  }
+
+  /**
+   * This method makes API call to K8 server to create configmap
+   *
+   * @param namespace
+   * @param v1ConfigMap
+   * @throws IOException
+   * @throws ApiException
+   */
+  private void createConfigmap(String namespace, V1ConfigMap v1ConfigMap)
+      throws IOException, ApiException {
+    Call call =
```

Comment:
plz rebase and make it reuse the following functions
```
Kubectl.create()
  .apiClient(..)
  .namespace(namespace)
  .resource(...)
  .execute()
```

Reply:
@yue9944882 - Can we access classes from `extended` in `util`?. My IDE is showing error without adding dependency in Util's pom.xml

Reply:
@yue9944882 and @brendandburns - Can you please help here?